docs/releases/0_10_0.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: 0.10.0
-sidebar_position: 29
+sidebar_position: 30
```

Comment:
I'm assuming you looked into it. Is there a better way to have things in reverse order?